frontend/packages/pipelines-plugin/locales/en/pipelines-plugin.json

```diff
@@ -437,11 +438,22 @@
   "Download all TaskRun logs": "Download all TaskRun logs",
   "TaskRun log": "TaskRun log",
   "Pod not found": "Pod not found",
+  "PipelineRun Name": "PipelineRun Name",
+  "Approvers": "Approvers",
+  "Task Name": "Task Name",
+  "Current Status": "Current Status",
+  "Approve": "Approve",
+  "Reject": "Reject",
+  "Approval status": "Approval status",
+  "ApprovalTasks": "ApprovalTasks",
+  "An error occurred. Please try again": "An error occurred. Please try again",
+  "<0>{type === 'approve'\n          ? 'Are you sure you want to approve'\n          : 'Please provide a reason for not approving'} <2></2><3>{name}</3> in <6></6><7></7><8>{pipelineRunName}</8>?</0>": "<0>{type === 'approve'\n          ? 'Are you sure you want to approve'\n          : 'Please provide a reason for not approving'} <2></2><3>{name}</3> in <6></6><7></7><8>{pipelineRunName}</8>?</0>",
```

Comment:
It doesn't look fine. The translation string contains conditions as well. Can you fix it.